src/clipper/argparser.py

```diff
@@ -690,6 +690,14 @@ def getArgParser(clipper_paths: ClipperPaths) -> argparse.ArgumentParser:
         default="yt_dlp",
         help="Choose a youtube_dl alternative for downloading videos.",
     )
+
+    parser.add_argument(
+      "--cookies",
```

Comment:
Since this should be a valid path to a cookies file, I think `cookiefile` would be a better name for the flag and it would be more consistent with the yt-dlp flag.